server/sonar-web/src/main/js/components/SourceViewer/SourceViewerBase.js

```diff
@@ -580,6 +587,14 @@ export default class SourceViewerBase extends React.PureComponent {
     }
   };
 
+  handleFilterLine = (line /*: SourceLine */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)